### ISSUE TITLE
ColorPickerView exception on startup when target SDK = 28: "IllegalArgumentException: x must be < bitmap.width()"

### ISSUE THREAD

**Issue by kiwiandroiddev** [skydoves/ColorPickerView#15]
If the compile and target SDK versions are changed to 28 in the sample app then this exception will be thrown during initial layout:
```kotlin
09-25 16:36:31.481 7552-7552/com.skydoves.colorpickerviewdemo E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.skydoves.colorpickerviewdemo, PID: 7552
    java.lang.IllegalArgumentException: x must be < bitmap.width()
        at android.graphics.Bitmap.checkPixelAccess(Bitmap.java:1780)
        at android.graphics.Bitmap.getPixel(Bitmap.java:1728)
        at com.skydoves.colorpickerview.ColorPickerView.getColorFromBitmap(ColorPickerView.java:211)
        at com.skydoves.colorpickerview.ColorPickerView.setSelectorPoint(ColorPickerView.java:356)
        at com.skydoves.colorpickerview.ColorPickerView.selectCenter(ColorPickerView.java:402)
        at com.skydoves.colorpickerview.ColorPickerView$1.onGlobalLayout(ColorPickerView.java:111)
        at android.view.ViewTreeObserver.dispatchOnGlobalLayout(ViewTreeObserver.java:945)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2357)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1460)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7183)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:949)
        at android.view.Choreographer.doCallbacks(Choreographer.java:761)
        at android.view.Choreographer.doFrame(Choreographer.java:696)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:935)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
``` 
Edit: reverting the targetSdkVersion to 27 and leaving the compileSdkVersion at 28 suppresses the issue. They must have introduced additional restrictions to `Bitmap.getPixel()` in Android 28

**Comment by skydoves** [comment]
I aware know this issue.
It will be fixed next version soon.
Thank you for your issue!

**Comment by kiwiandroiddev** [comment]
Awesome, thank you :)

**Comment by skydoves** [comment]
It's fixed at version 2.0.1!
Thank you!

**Comment by savejraees** [comment]
still not fixed upto 2.2.3 version
